### PR TITLE
add custom buckets for latency

### PR DESCRIPTION
**Pull request by chrfla-cognite**
changing unit for histograms from seconds to minutes, so it can also cover larger upload queues.

### CODE REVIEW

**Review by pavelzubarev — COMMENTED** [review]
I'm not entirely sure that solves your problem. Would it be simpler to observe minutes and not seconds for each latency metric we have?

File: cognite/extractorutils/uploader.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ class TimestampedObject:
 TIMESERIES_UPLOADER_QUEUE_SIZE = Gauge("cognite_timeseries_uploader_queue_size", "Internal queue size")
 
 TIMESERIES_UPLOADER_LATENCY = Histogram(
-    "cognite_timeseries_uploader_latency", "Distribution of times in seconds records spend in the queue"
+    "cognite_timeseries_uploader_latency", "Distribution of times in seconds records spend in the queue",
```
Comment (pavelzubarev) [inline cognite/extractorutils/uploader.py:223]: minutes?

File: cognite/extractorutils/uploader.py [hunk 2]
```diff
@@ -234,7 +234,7 @@ class TimestampedObject:
 SEQUENCES_UPLOADER_QUEUE_SIZE = Gauge("cognite_sequences_uploader_queue_size", "Internal queue size")
 
 SEQUENCES_UPLOADER_LATENCY = Histogram(
-    "cognite_sequences_uploader_latency", "Distribution of times in seconds records spend in the queue"
+    "cognite_sequences_uploader_latency", "Distribution of times in seconds records spend in the queue",
```
Comment (pavelzubarev) [inline cognite/extractorutils/uploader.py:237]: minutes?

File: cognite/extractorutils/uploader.py [hunk 3]
```diff
@@ -244,7 +244,7 @@ class TimestampedObject:
 EVENTS_UPLOADER_QUEUE_SIZE = Gauge("cognite_events_uploader_queue_size", "Internal queue size")
 
 EVENTS_UPLOADER_LATENCY = Histogram(
-    "cognite_events_uploader_latency", "Distribution of times in seconds records spend in the queue"
+    "cognite_events_uploader_latency", "Distribution of times in seconds records spend in the queue",
```
Comment (pavelzubarev) [inline cognite/extractorutils/uploader.py:247]: minutes?

File: cognite/extractorutils/uploader.py [hunk 4]
```diff
@@ -254,7 +254,7 @@ class TimestampedObject:
 FILES_UPLOADER_QUEUE_SIZE = Gauge("cognite_files_uploader_queue_size", "Internal queue size")
 
 FILES_UPLOADER_LATENCY = Histogram(
-    "cognite_files_uploader_latency", "Distribution of times in seconds records spend in the queue"
+    "cognite_files_uploader_latency", "Distribution of times in seconds records spend in the queue",
```
Comment (pavelzubarev) [inline cognite/extractorutils/uploader.py:257]: minutes?

**Review by pavelzubarev — COMMENTED** [review]
fix metrics description

File: cognite/extractorutils/uploader.py (hunk 1 above)
Comment (chrfla-cognite) [reply cognite/extractorutils/uploader.py]: done

File: cognite/extractorutils/uploader.py (hunk 2 above)
Comment (chrfla-cognite) [reply cognite/extractorutils/uploader.py]: done

File: cognite/extractorutils/uploader.py (hunk 3 above)
Comment (chrfla-cognite) [reply cognite/extractorutils/uploader.py]: done

File: cognite/extractorutils/uploader.py (hunk 4 above)
Comment (chrfla-cognite) [reply cognite/extractorutils/uploader.py]: done

**Review by pavelzubarev — APPROVED** [review]

**Comment by chrfla-cognite** [comment]
> I'm not entirely sure that solves your problem. Would it be simpler to observe minutes and not seconds for each latency metric we have?

yep, good point. It's switched to minutes now

**Comment by mathialo** [comment]
@pavelzubarev I assume your 👍 means that this won't break your dashboards?

**Review by mathialo — APPROVED** [review]

**Comment by pavelzubarev** [comment]
> @pavelzubarev I assume your 👍 means that this won't break your dashboards?

It won't brake it... we just change the bucket resolution, which could affect alerts based on that, but there is no alerts yet. so all is good

**Pull request merged**